README.md

```diff
@@ -74,7 +74,15 @@ To run the tests for particular proposals,
 ./runTestImages.ts --match upgrade
 ```
 
-## Contributing
+## Debugging
+
+To get the local files into the container, use a mount. E.g.
```

Comment:
I think the terminology is...

```suggestion
To get the local files into the container, use a volume. E.g.
```

or perhaps change "use a mount" to "mount a volume"

Reply:
It's really not a "volume", but [bind mount](https://docs.docker.com/storage/bind-mounts/). I'll use the full term.